floris/tools/floris_interface.py

```diff
@@ -401,10 +400,7 @@ def calculate_horizontal_plane(
 
         # Compute the cutplane
         horizontal_plane = CutPlane(
-            df,
-            self.floris.grid.grid_resolution[0],
-            self.floris.grid.grid_resolution[1],
-            "z"
+            df, self.floris.grid.grid_resolution[0], self.floris.grid.grid_resolution[1], "z"
```

Comment:
In the case of many arguments to a function, the preferred style is to list each on a new line. This enables easily eyeballing the inputs and swapping out one for another. If you add a trailing comma to the list of arguments, ruff will leave it as is.

Reply:
This was possibly a ruff format fix? I've now reorganized as you've suggested